bc_obps/reporting/service/sync_validation_service.py

```diff
@@ -0,0 +1,47 @@
+from reporting.models import ReportVersion
+from service.reporting_year_service import ReportingYearService
+
+
+class SyncValidationService:
+    """
+    Service to validate whether sync operations are allowed for a given report version.
+    """
+
+    @classmethod
+    def is_sync_allowed(cls, version_id: int) -> bool:
+        """
+        Check if syncing with admin data is allowed for the given report version.
+
+        Sync is NOT allowed if:
+        1. The report is not for the current reporting year
+        2. The operation is not currently owned by the operator who filed the report
+
+        Args:
+            version_id: The report version ID
+
+        Returns:
+            bool: True if sync is allowed, False otherwise
+        """
+        try:
+            report_version = ReportVersion.objects.select_related(
+                'report__operation', 'report__operator', 'report__reporting_year'
+            ).get(id=version_id)
+        except ReportVersion.DoesNotExist:
+            return False
+
```

Comment:
Whoever calls this service with a non-existent report version should be rewarded with a `DoesNotExist` error, making it disappear here only hides a potential issue.
I'd remove the try/except block entirely

```suggestion
        report_version = ReportVersion.objects.select_related(
            'report__operation', 'report__operator', 'report__reporting_year'
        ).get(id=version_id)
```